contracts/ERC1155NFT.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ERC1155NFT is ERC1155, Ownable {
+
+    mapping(uint256 => uint256) private balances;
+
+    mapping(uint256 => address[]) private owners;
+
+    constructor(string memory _uri) ERC1155(_uri) {}
+
+    function setURI(string memory newuri) public onlyOwner {
+        _setURI(newuri);
+    }
+
+    function mint(
+        address account,
+        uint256 id,
+        uint256 amount,
+        bytes memory data
+    ) public payable onlyOwner {
+        _mint(account, id, amount, data);
+        balances[id] += amount;
+        for (uint256 i = 0; i < owners[id].length; i++) {
+            if (owners[id][i] == account) {
+                return;
+            }
+        }
+        owners[id].push(account);
+    }
+
+    function mintBatch(
```

Comment:
I think we agreed to remove batch minting as it does not add any extra value for us atm?

Reply:
Yah its not in use now, will remove it.

Reply:
Solved